decidim-meetings/spec/commands/admin/publish_meeting_spec.rb

```diff
@@ -40,16 +40,29 @@ module Admin
             expect(action_log.version).to be_present
           end
 
-          it "schedules a upcoming meeting notification job 48h before start time" do
-            expect(UpcomingMeetingNotificationJob)
-              .to receive(:generate_checksum).and_return "1234"
+          context "when it's a future meeting" do
```

Comment:
Seems like it's the same spec 3 times with minimal differences. Can you move it to a shared example 🙏🏽?